justega247/favorite-things
# #167343183: Audit user favourite things

#### What does this PR do?
- This PR is for the implementation for retrieving the audit log for a user's favourite thing instance
#### Description of Task to be completed?
- create the implementation for tracking the audit log using django-simple-history
- write tests to ensure implementation works as expected
#### How should this be manually tested?
- Start the application by going to the root directory and running **python manage.py runserver** 
- Navigate to the endpoint **http://127.0.0.1:8000/api/v1/favorite/history/:id** and make a GET request
- You should see the audit log of the favourite thing instance
#### What are the relevant pivotal tracker stories?
[#167343183](https://www.pivotaltracker.com/story/show/167343183)
